modules/cluster-autoscaler-machineset.adoc

```diff
@@ -0,0 +1,89 @@
+// Module included in the following assemblies:
+//
+// * machine_management/applying-autoscaling.adoc
+// * post_installation_configuration/cluster-tasks.adoc
+
+[id="cluster-autoscaler-machineset{context}"]
+= Configuring the machineset
+
+The MachineSet which creates the Windows Machines requires configuring a file template. On creation, a `windows-user-data` secret is created. On subsequent use, the `windows-user-data` may be used by the MachineSets. The associated machinesets may have the following labels:
+
+* `machine.openshift.io/os-id: Windows`
+* `machine.openshift.io/cluster-api-machine-role: worker`
+* `machine.openshift.io/cluster-api-machine-type: worker`
+
+The following label has to be added to the Machine spec within the MachineSet spec:
+
+* `node-role.kubernetes.io/worker:``
++
+[IMPORTANT]
+====
+If these labels are not added, the the Windows node will not be marked as a `worker` node.
+====
+
+.Procedure
+
+. Configure the machineset template with values from your environment:
+
+[source, yaml]
+----
+apiVersion: machine.openshift.io/v1beta1
+kind: MachineSet
+metadata:
+  labels:
+    machine.openshift.io/cluster-api-cluster: <infrastructureID> <1>
+  name: winworker
+  namespace: openshift-machine-api
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      machine.openshift.io/cluster-api-cluster: <infrastructureID>
+      machine.openshift.io/cluster-api-machineset: winworker
+  template:
+    metadata:
+      labels:
+        machine.openshift.io/cluster-api-cluster: <infrastructureID>
+        machine.openshift.io/cluster-api-machine-role: worker
+        machine.openshift.io/cluster-api-machine-type: worker
+        machine.openshift.io/cluster-api-machineset: winworker
+        machine.openshift.io/os-id: Windows
+    spec:
+      metadata:
+        labels:
+          node-role.kubernetes.io/worker: ""
+      providerSpec:
+        value:
+          apiVersion: vsphereprovider.openshift.io/v1beta1
+          credentialsSecret:
+            name: vsphere-cloud-credentials
+          diskGiB: 128
+          kind: VSphereMachineProviderSpec
+          memoryMiB: 16384
+          metadata:
+            creationTimestamp: null
+          network:
+            devices:
+            - networkName:  "<VM Network Name>" <2>
+          numCPUs: 4
+          numCoresPerSocket: 1
+          snapshot: ""
+          template: <Windows_VM_template> <3>
+          userDataSecret:
+            name: worker-user-data-managed
+          workspace:
+             datacenter: <vCenter DataCenter Name> <4>
+             datastore: <vCenter Datastore Name> <5>
+             folder: <Path to VM Folder in vCenter> # e.g. /DC/vm/ocp45-2tdrm <6>
+             server: <vCenter Server FQDN/IP> <7>
+----
+
+<1> The <infrastructureID> tag should be replaced with the output of the following command:
++
+`$ oc get -o jsonpath='{.status.infrastructureName}{"\n"}' infrastructure cluster`
```

Comment:
I would appreciate someone providing an example output so that I can include that our users.